src/Compatibility/ApiDiff/Microsoft.DotNet.ApiDiff.Tool/Program.cs

```diff
@@ -0,0 +1,209 @@
+            Arity = ArgumentArity.ExactlyOne,
+            IsRequired = false
+        };
+
+        Option<string> optionAfterAssembliesFolderPath = new(["--after", "-a"])
+        {
+            Description = "The path to the folder containing the new (after) assemblies.",
+            Arity = ArgumentArity.ExactlyOne,
+            IsRequired = false
+        };
+
+        Option<string> optionAfterRefAssembliesFolderPath = new(["--refafter", "-ra"])
+        {
+            Description = "The path to the folder containing the new (after) reference assemblies.",
+            Arity = ArgumentArity.ExactlyOne,
+            IsRequired = false
+        };
+
+        Option<string> optionOutputFolderPath = new(["--output", "-o"])
+        {
+            Description = "The path to the output folder.",
+            Arity = ArgumentArity.ExactlyOne,
+            IsRequired = true
+        };
+
+        Option<string> optionBeforeFriendlyName = new(["--beforeFriendlyName", "-bfn"])
+        {
+            Description = "The friendly name to describe the 'before' assembly.",
+            Arity = ArgumentArity.ExactlyOne,
+            IsRequired = true
+        };
+
+        Option<string> optionAfterFriendlyName = new(["--afterFriendlyName", "-afn"])
+        {
+            Description = "The friendly name to describe the 'after' assembly.",
+            Arity = ArgumentArity.ExactlyOne,
+            IsRequired = true
+        };
+
+        Option<string> optionTableOfContentsTitle = new(["--tableOfContentsTitle", "-tc"])
+        {
+            Description = "The title of the markdown table of contents file that is placed in the output folder.",
+            Arity = ArgumentArity.ExactlyOne,
+            IsRequired = true
+        };
+
+        Option<string[]?> optionAssembliesToExclude = new(["--assembliesToExclude", "-eas"], () => null)
+        {
+            Description = "Assemblies to exclude from the diff.",
+            Arity = ArgumentArity.ZeroOrMore,
+            IsRequired = false,
+        };
+
+        Option<string[]?> optionAttributesToExclude = new(["--attributesToExclude", "-eattrs"], () => null)
+        {
+            Description = "Attributes to exclude from the diff.",
+            Arity = ArgumentArity.ZeroOrMore,
+            IsRequired = false
+        };
+
+        Option<string[]?> optionApisToExclude = new(["--apisToExclude", "-eapis"], () => null)
+        {
+            Description = "APIs to exclude from the diff.",
+            Arity = ArgumentArity.ZeroOrMore,
+            IsRequired = false
+        };
+
+        Option<bool> optionAddPartialModifier = new(["--addPartialModifier", "-apm"], () => false)
+        {
+            Description = "Add the 'partial' modifier to types."
+        };
+
+        Option<bool> optionHideImplicitDefaultConstructors = new(["--hideImplicitDefaultConstructors", "-hidc"], () => false)
+        {
+            Description = "Hide implicit default constructors from types."
```

Comment:
Should this be an option?  I would expect we'd always want to show when a change removed a default constructor, or added one.  When would someone not want to see this?

Reply:
I remember api diffs where area owners were wondering why a default constructor was being shown when no one had added one. But we still decided that we needed to keep showing them.

I suppose it's similar to api-docs, where we always show a default constructor even if one isn't explicitly defined in source code. I'll remove this as an option, since I agree with what you said that we probably will never disable them.